pkg/integrations/node_exporter/collectors.go

```diff
@@ -0,0 +1,188 @@
+package node_exporter // nolint:golint
```

Comment:
just curious, what are these golint tags about?

Reply:
`golint` complains about the package having an underscore in the name, but I find that having an underscore is appropriate here